django/db/models/indexes.py

```diff
@@ -44,21 +44,32 @@ def check_name(self):
             self.name = 'D%s' % self.name[1:]
         return errors
 
-    def create_sql(self, model, schema_editor, using=''):
+    def get_sql_create_params(self, model, schema_editor, using=''):
```

Comment:
`params` mostly refers to the parameters passed to `cursor.execute()` and are the values that are escaped and injected by the database (backend) at the very last before executing the query. Could you rename the method to something like `get_sql_create_template_values` if that makes sense?